src/runtime_src/core/pcie/windows/shim.cpp

```diff
@@ -1206,6 +1233,14 @@ get_bdf_info(xclDeviceHandle hdl, uint16_t bdf[3])
   shim->get_bdf_info(bdf);
 }
 
+void
+get_mailbox_info(xclDeviceHandle hdl, xcl_mailbox* value)
+{
+  xrt_core::message::send(xrt_core::message::severity_level::debug, "XRT", "mailbox_info()");
+  shim* shim = get_shim_object(hdl);
+  shim->get_mailbox_info(value);
```

Comment:
```suggestion
  get_shim_object(hdl)->get_mailbox_info(value);
```
Probably even better if `get_shim_object(hdl)` was returning a reference instead of a pointer

Reply:
I am just following the same syntax of already existing piece of code from the same file "shim.cpp"

Reply:
Yeah this is legacy not to be changed now. The intend is that get_shim_object will check the validity of the pointer and return nullptr if invalid, which of course must be checked. Unfortunately this check is not wired up on windows.
Maybe at least save return type in an auto, which half the calls do in this file and the remaining half have escaped my review.